src/query/builder.js

```diff
@@ -118,6 +123,21 @@ assign(Builder.prototype, {
     return this;
   },
 
+  // With Recursive
+  // ------
+
+  withRecursive(alias, statement) {
+    validateWithArgs(alias, statement, 'withRecursive');
+    return this.withRecursiveWrapped(alias, statement);
+  },
+
+  // Helper for compiling any advanced `withRecursive` queries.
+  withRecursiveWrapped(alias, query) {
```

Comment:
why separate this from `withRecursive` if it just passes same params after validation? Also, should it be public?

Reply:
I'm just mirroring the pattern established by `with` and `withWrapped`.